packages/file-asset-apis/test/lib/chunked-file-sender-spec.ts

```diff
@@ -116,89 +126,105 @@ describe('ChunkedSlicer', () => {
     describe('file destination names', () => {
         it('can make correct base paths', async () => {
             const test = new Test(FileSenderType.file, makeConfig());
-
+            // @ts-expect-error
             expect(await test.createFileDestinationName(path)).toEqual(`${path}/${workerId}`);
             expect(test.verifyCalled).toEqual(true);
         });
 
         it('can make correct path', async () => {
             const newPath = `${path}/final/dir`;
             const test = new Test(FileSenderType.file, makeConfig());
-
+            // @ts-expect-error
             expect(await test.createFileDestinationName(newPath)).toEqual(`${newPath}/${workerId}`);
             expect(test.verifyCalled).toEqual(true);
         });
 
         it('can add extensions', async () => {
             const test = new Test(FileSenderType.file, makeConfig({ extension: 'stuff' }));
-
+            // @ts-expect-error
```

Comment:
Why are there ts-expect-errors added all over this file?

Reply:
Since we are exporting these classes, I added additional restrictions on many of the methods as they are meant to be used by the child class internally or only by the parent class internally. This test file was meant to test the abstract class and its behaviours.

Reply:
I found often it is more correct to make them public. Tests should rarely be using private functions since it will make them more difficult to refactor and modify. In this case, it is useful to make it public since we don't know if a developer may want to use the isolated method and since we are testing it we are ensuring its contract doesn't break.